src/services/notifications.service.js

```diff
@@ -434,15 +448,39 @@ const generateEmailsToSend = (registration, lcContactConfig, configData) => {
         case "FD_FB":
           templateID = configData.notify_template_keys.fd_feedback;
           break;
+        case "RNG_PENDING":
+          templateID = cy
+            ? RNG_PENDING_TEMPLATE_ID_CY
```

Comment:
Is there a reason why these don't come from config like the other templates IDs?

Reply:
This was agreed with Simon for the purpose of simplification. (There should be no changes to this template, so don't need versioning it)